src/PresentationalComponents/SystemsTable/createColumns.js

```diff
@@ -13,23 +10,6 @@ export const createColumns = (defaultColumns, columns) =>
         : {
             ...column,
             ...correspondingColumn,
-            ...(column.key === 'groups'
-              ? {
-                  renderFunc: (groups) =>
-                    isEmpty(groups) ? (
```

Comment:
If there is no group, we need to show N/A. With this change, I believe this will not be the case ?

Reply:
@mkholjuraev, it will be already done by Inventory filter config (the default one). The removed code was making sure that we render the group names as links - this is no longer required.

Reply:
You also can run it locally and see that N/A is rendered for empty groups.